runtime/onert/core/include/ir/train/ITrainableOperation.h

```diff
@@ -28,7 +28,9 @@ namespace train
 
 struct TrainableOperationVisitor;
 
-class ITrainableOperation : public IOperation
+// NOTE Virtual inheritance is introduced because trainable operations inherit
+//      `ITrainableOperation` and `Operation` which inherit `IOperation`).
```

Comment:
```suggestion
// NOTE Virtual inheritance is introduced because trainable operations inherit
//      `ITrainableOperation` and `Operation` which inherit `IOperation`.
```

Reply:
I fixed it.